gazebo/sensors/Sensor.hh

```diff
@@ -235,6 +239,11 @@ namespace gazebo
       ///        Sensor::SetUpdateRate.
       protected: common::Time updatePeriod;
 
+      /// \brief Whether to enforce strict sensor update rate, even if physics
+      ///        time has to slow down to wait for sensor updates to satisfy
+      ///        the desired rate.
+      protected: bool useStrictRate;
```

Comment:
actually, this may have broken ABI

Reply:
never mind, this didn't go to a release branch, and I think we already talked about it